src/content/configuration/dev-server.mdx

```diff
@@ -1332,7 +1333,7 @@ module.exports = {
 
 `boolean` `string` `[string]` `object` `[object]`
 
-This options allows to configure options for serving static files from directory (by default 'public' directory). To disable set it to `false`:
+This option allows configuring options for serving static files from the directory (by default 'public' directory). To disable set it to `false`:
```

Comment:
What's wrong with `allows to configure`?

Reply:
Nothing wrong with allows to configure, this whole sentence is in present continuos tense and allows configuring seems to fit in here that why i did that